package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "memoize-one": "^5.0.0",
-    "prop-types": "^15.6.0",
+    "prop-types": "^15.6.2",
```

Comment:
Besides, this change does not make sense because the prop-types version in package.json has a caret (it means that only the major number is locked). Hence, just doing a `npm install`, the prop-types version is upgraded to 15.8.1

Reply:
Right, running `npm install` is enough. I am just explicitly updating it here since 15.6.0 and 15.6.1 still have the vulnerability and it would seem a bit strange to me to keep it here, even with the caret